sdk/core/azure-core-http-netty/src/main/java/module-info.java

```diff
@@ -3,14 +3,16 @@
 
 module com.azure.http.netty {
     requires transitive com.azure.core;
-    requires reactor.netty;
     requires io.netty.buffer;
     requires io.netty.common;
     requires io.netty.transport;
     requires io.netty.handler;
     requires io.netty.codec;
     requires io.netty.codec.http;
     requires io.netty.handler.proxy;
+    requires reactor.netty;
+    requires reactor.netty.core;
+    requires reactor.netty.http;
 
     exports com.azure.core.http.netty;
     exports com.azure.core.http.netty.implementation;       // FIXME this should not be a long-term solution
```

Comment:
Can this be removed or narrowed down to export to specific modules only? `exports ... to ...`

Reply:
For me, the best path forward to this would be moving the SPI into the root package and removing the `exports ... implementation` completely.

cc: @JonathanGiles

Reply:
Lets do that in a separate PR so we can see what we are dealing with, but I am fine in general with Alan's preferred approach.

Reply:
Tracking issue https://github.com/Azure/azure-sdk-for-java/issues/18445